### DISCUSSION TITLE
How are Catena-X, Tractus-X, IDS, Gaia-X & EDC related?

### DISCUSSION THREAD

**Discussion by nitin-vavdiya** [eclipse-edc/Connector#1016] [Q&A]
If we refer to any online available material related to EDC, we can find buzz words like **Catenax**, **Tractusx**, **international data spaces(IDS)**, **Eclipse dataspace connector(EDC)**, **Gaia-x**, etc.

How are these terms/things related to each other?

**Comment by mspiekermann** [comment]
- *Catena-X*: Uses the EDC as Connector component to build a dataspace between participants
- *Gaia-X*: EDC is an implementation to allow organizations to exchange data compliant to rules and policies of the Gaia-X AISBL according to the corresponding Trust Framework
- *Tractus-X*: Is an Eclipse Foundation OSS project initiated by Catena-X consortia, where specific (Catena-X related) EDC-extensions can be implemented under clear governance and rules by the EF.
- *IDS*:  EDC is an implementation to allow organizations to exchange data compliant to rules and policies of the IDSA according to the corresponding Reference Architecture Model (RAM) and Rule Book

Please have an additional look at https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/discussions/1037, and included links for a better overview and information.
